src/inc_430.a51

```diff
@@ -14,22 +14,26 @@
 ;    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 CONFIG_CHAN_COUNT       EQU    031h
-CONFIG_SHIFT_LO         EQU    080h
+
+;CONFIG_SHIFT_LO         EQU    060h			; Shift = 7.6Mhz (Germany)
+;CONFIG_SHIFT_HI         EQU    002h
+CONFIG_SHIFT_LO         EQU    080h				; Shift = 1.6Mhz (France)
 CONFIG_SHIFT_HI         EQU    000h
+
 CONFIG_PLL_DIV_LO       EQU    090h
 CONFIG_PLL_DIV_HI       EQU    020h
 CONFIG_SCAN_DURATION	EQU    008h
 
 freq_list:
-        ; Channel : 0 Frequency : 430.025000
-        DB      086h
-        DB      062h
-        ; Channel : 1 Frequency : 430.050000
-        DB      086h
-        DB      064h
-        ; Channel : 2 Frequency : 430.075000
-        DB      086h
-        DB      066h
+        ; Channel : 0 Frequency : 439.375000 (DB0HM)
+        DB      089h
+        DB      04Eh
+        ; Channel : 1 Frequency : 439.425000 (DB0GK)
+        DB      089h
+        DB      052h
+        ; Channel : 2 Frequency : 439.837500 (DO0DAP, needs Subtone :-( ) 
+        DB      089h
+        DB      073h
```

Comment:
yes, I will do. Before I have a question. The 70cm Band in Germany is from 430Mhz to 440Mhz.
Is my understanding correct, that at France the band stops at 435Mhz?
So the question is shall we extend the frequency for the Germany band or shall we leave as it was before (your version) and I will build a special version for my needs (or only do my configuration via EEPROM).
Just give me your preferred  solution

Reply:
We got also the 430-440 band. I don't remember why i have not implemented this channel. You can freely add them

Reply:
may be because it's a lot of effort :-). 
Why was the shift activated by default, for most of your channels. Does this make sense or should I set a "0" as default for channel 0..30, also?